pyproject.toml

```diff
@@ -0,0 +1,19 @@
+[build-system]
+requires = ["setuptools >= 61", "setuptools_scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "league-rpc-linux"
+dynamic = ["version"]
+requires-python = ">= 3.8"
```

Comment:
Python 3.8 will not work since "Match & Case" syntax is being used
I was using python >3.11 while developing, so I think it's safest to use that.. potentially 3.10

Pretty sure match statements came to python in 3.10

Reply:
![image](https://github.com/Its-Haze/league-rpc-linux/assets/90850836/04690c88-3946-4d56-b87b-1b07288ba241)

Reply:
I didn't know what's the actual required Python version, it was basically random choice because only Python 3.8+ gained proper `build` support. Updated now to 3.10

Reply:
I'm pretty sure that 3.10 is going to work. but as a QA i must test. so i will double check with 3.10 and see that everything runs as expected.. will return back when i know how it goes.